shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profile/orders/presentation/OrderHistoryComponent.kt

```diff
@@ -111,13 +111,11 @@ open class OrderHistoryComponent(
             is OrderHistoryEvents.FetchContents -> fetchOrderHistory()
             is OrderHistoryEvents.PrimaryButtonPressed -> when(event.order.status) {
                 OrderStatus.PENDING -> commonUiState.update { it.copy(confirmAction = OrderHistoryConfirmAction.Cancellation(event.order)) }
```

Comment:
While testing, I noticed that there is an issue preventing me from canceling an order on the Order History screen

Reply:
Fixed :)